Assets/MixedRealityToolkit.Extensions/PhotoCapture/CameraIntrinsics.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.PhotoCapture
+{
+    public class CameraIntrinsics
+    {
+        //
```

Comment:
this formatting will likley not be picked up by the doc generator. please consider reformatting

Reply:
I'm going to take another pass on PhotoCapture summaries in a seaprate review. To unblock some additional sharing work, I'm going to go ahead and check this in.